packages/utils/src/index.ts

```diff
@@ -1,540 +1,704 @@
+  logger as logger_imported,
+  makeDsn as makeDsn_imported,
+  makeFifoCache as makeFifoCache_imported,
+  makePromiseBuffer as makePromiseBuffer_imported,
+  markFunctionWrapped as markFunctionWrapped_imported,
+  maybeInstrument as maybeInstrument_imported,
+  memoBuilder as memoBuilder_imported,
+  node as node_imported,
+  nodeStackLineParser as nodeStackLineParser_imported,
+  normalize as normalize_imported,
+  normalizePath as normalizePath_imported,
+  normalizeToSize as normalizeToSize_imported,
+  normalizeUrlToBase as normalizeUrlToBase_imported,
+  objectify as objectify_imported,
+  originalConsoleMethods as originalConsoleMethods_imported,
+  parseBaggageHeader as parseBaggageHeader_imported,
+  parseEnvelope as parseEnvelope_imported,
+  parseRetryAfterHeader as parseRetryAfterHeader_imported,
+  parseSemver as parseSemver_imported,
+  parseStackFrames as parseStackFrames_imported,
+  parseUrl as parseUrl_imported,
+  propagationContextFromHeaders as propagationContextFromHeaders_imported,
+  rejectedSyncPromise as rejectedSyncPromise_imported,
+  relative as relative_imported,
+  resetInstrumentationHandlers as resetInstrumentationHandlers_imported,
+  resolve as resolve_imported,
+  resolvedSyncPromise as resolvedSyncPromise_imported,
+  safeJoin as safeJoin_imported,
+  serializeEnvelope as serializeEnvelope_imported,
+  severityLevelFromString as severityLevelFromString_imported,
+  snipLine as snipLine_imported,
+  stackParserFromStackParserOptions as stackParserFromStackParserOptions_imported,
+  stringMatchesSomePattern as stringMatchesSomePattern_imported,
+  stripSentryFramesAndReverse as stripSentryFramesAndReverse_imported,
+  stripUrlQueryAndFragment as stripUrlQueryAndFragment_imported,
+  supportsDOMError as supportsDOMError_imported,
+  supportsDOMException as supportsDOMException_imported,
+  supportsErrorEvent as supportsErrorEvent_imported,
+  supportsFetch as supportsFetch_imported,
+  supportsHistory as supportsHistory_imported,
+  supportsNativeFetch as supportsNativeFetch_imported,
+  supportsReferrerPolicy as supportsReferrerPolicy_imported,
+  supportsReportingObserver as supportsReportingObserver_imported,
+  timestampInSeconds as timestampInSeconds_imported,
+  triggerHandlers as triggerHandlers_imported,
+  truncate as truncate_imported,
+  updateRateLimits as updateRateLimits_imported,
+  urlEncode as urlEncode_imported,
+  uuid4 as uuid4_imported,
+  validSeverityLevels as validSeverityLevels_imported,
+  vercelWaitUntil as vercelWaitUntil_imported,
+  watchdogTimer as watchdogTimer_imported,
+  winterCGHeadersToDict as winterCGHeadersToDict_imported,
+  winterCGRequestToRequestData as winterCGRequestToRequestData_imported,
 } from '@sentry/core';
 
-export type {
-  InternalGlobal,
-  PromiseBuffer,
-  RateLimits,
-  AddRequestDataToEventOptions,
-  SdkSource,
-  // eslint-disable-next-line deprecation/deprecation
-  TransactionNamingScheme,
+/** @deprecated Import from `@sentry/node` instead. */
```

Comment:
h: I think all of these should mention `@sentry/core` instead of `@sentry/node`? 😅

Reply:
brain = fried

Reply:
As long as stuff is easily search-replaced nothing is lost :D